app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/logging/AndroidLogcatLoggerRegistrar.kt

```diff
@@ -47,8 +45,8 @@ class AndroidLogcatLoggerRegistrar @Inject constructor(
     }
 
     override fun onVpnProcessCreated() {
-        if (appBuildConfig.isDebug || appTpLocalFeature.verboseLogging().isEnabled()) {
-            LogcatLogger.install(AndroidLogcatLogger(LogPriority.DEBUG))
+        if (appTpLocalFeature.verboseLogging().isEnabled()) {
```

Comment:
@karlenDimla Did we remove the `appBuildConfig.isDebug` on purpose?
I believe this changes the behavior. Previously, the logger was installed on all debug builds or any builds with the local flag enabled. Now, it is installed only if the local flag is enabled.